site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Contribution budget'
+subhead: >
+  Learn about the role of the contribution budget and how to allocate it to capture the data you need.  
+description: >
+  Learn about the role of the contribution budget and how to allocate it to capture the data you need.  
+date: 2022-03-01
+updated: 2023-03-14
+authors:
+  - maudn
+---
+
+To protect user privacy, individual users' contributions must have an upper limit.
```

Comment:
Could we add a one-liner that specifies the context i.e. summary reports in Attribution Reporting?

Reply:
done